permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -48,16 +55,102 @@ public static PermisoDialogFragment newInstance(@Nullable String title, @NonNull
         return dialogFragment;
     }
 
+    public static class Builder {
+        private int titleId = 0;
+        private String title = null;
+        private int buttonTextId = 0;
+        private String message = null;
+        private int messageId = 0;
+        // by default, interpret the dialog msg body as string text
+        private boolean interpretHtml = false;
+        private String buttonText = null;
+
+
+        private Builder() {}
+
+        public Builder(int titleId, int msgBody, int buttonTextId) {
+            this();
+            this.titleId = titleId;
+            this.messageId = msgBody;
+            this.buttonTextId = buttonTextId;
+        }
+
+
+        public int getTitleId() { return titleId; }
+        public String getTitle() { return title; }
+        public String getMessage() { return message; }
+        public int getMessageId() { return messageId;}
+        public int getButtonTextId() { return buttonTextId; }
+        public String getButtonText() { return buttonText; }
+        public boolean isHtml() { return interpretHtml; }
+
+        public Builder setTitleId(int val) { titleId = val; return this; }
```

Comment:
We can just overload ```setTitle()```. No need to suffix the method name it with ```Id```. Same for the other methods.